tests/test_free_energy.py

```diff
@@ -260,26 +258,6 @@ def hif2a_ligand_pair_single_topology_lam0_state():
     return state
 
 
-@given(integers(1, 100), integers(1, 100))
-@seed(2023)
-@example(10, 3)
-@example(10, 1)
-@example(1, 10)
-def test_sample_max_buffer_frames(hif2a_ligand_pair_single_topology_lam0_state, n_frames, max_buffer_frames):
-    md_params = MDParams(n_frames, 1, 1, 2023)
-    frames_ref, _, _ = sample(hif2a_ligand_pair_single_topology_lam0_state, md_params)
-    frames_test, _, _ = sample(
-        hif2a_ligand_pair_single_topology_lam0_state, md_params, max_buffer_frames=max_buffer_frames
-    )
-
-    assert isinstance(frames_ref, np.ndarray)
-    assert isinstance(frames_test, StoredArrays)
-    assert len(frames_ref) == len(frames_test)
-
-    for frame_ref, frame_test in zip(frames_ref, frames_test):
-        np.testing.assert_array_equal(frame_ref, frame_test)
-
-
```

Comment:
This was testing for consistency when we pass `max_buffer_frames` vs. when we don't; it is no longer needed now that `max_buffer_frames` is required.